examples/lib/stories/widgets/overlay.dart

```diff
@@ -1,82 +1,64 @@
 import 'package:dashbook/dashbook.dart';
+import 'package:flame/components.dart';
 import 'package:flame/game.dart';
 import 'package:flame/input.dart';
-import 'package:flame/palette.dart';
 import 'package:flutter/material.dart';
 
-Widget overlayBuilder(DashbookContext ctx) {
-  return const OverlayExampleWidget();
+Widget _pauseMenuBuilder(BuildContext buildContext, ExampleGame game) {
```

Comment:
Our overlay example had some wrapper around it which made only this example look a bit different from the others.

I've changed it to be simpler and more consistent to out other examples.

I have also used it to test the case of overlay addition, which also caused the bug.